dts/bindings/rng/st,stm32-rng.yaml

```
@@ -16,3 +16,15 @@ properties:
 
     clocks:
       required: true
+
+    htcfg1:
+      type: int
+      required: false
+      description: |
+           Magic Number to be written to HTCR register prior to real configuration,
+           if any.
+
+    htcfg:
+      type: int
+      required: false
+      description: HTCR configuration
```

```suggestion
    health-test-config:
      type: int
      required: false
      description: |
          Health Test confuguration, blabla
```
Please explain when and why this property should be provided.

---

Done